fileutil/perm_other.go

```diff
@@ -0,0 +1,43 @@
+// +build !windows
```

Comment:
missing copyright notice (put 2017, and if it's a new file mostly written by you, your name - maybe we could eventually switch to an "sh authors" like Go)